core/domain/feedback_services.py

```diff
@@ -261,3 +263,40 @@ def enqueue_feedback_message_email_task(user_id):
     taskqueue_services.enqueue_task(
         feconf.FEEDBACK_MESSAGE_EMAIL_HANDLER_URL, {'user_id': user_id},
         feconf.DEFAULT_FEEDBACK_MESSAGE_EMAIL_COUNTDOWN_SECS)
+
+
+def get_feedback_message_references(user_id):
+    model = feedback_models.UnsentFeedbackEmailModel.get(user_id, strict=True)
+
+    return [feedback_domain.FeedbackMessageReferences(
+        reference['exploration_id'], reference['thread_id'],
+        reference['message_id'])
+            for reference in model.feedback_message_references]
+
+def update_feedback_message_reference(user_id, reference):
+    """Creates a new instance of UnsentFeedbackEmailModel or update existing
+    model instance for sending feedback message email."""
+
+    model = feedback_models.UnsentFeedbackEmailModel.get(user_id, strict=False)
+
+    if model is not None:
+        model.feedback_message_references.append(reference.to_dict())
+        model.put()
+
```

Comment:
Optional: No need for this newline (otherwise the else block looks a bit divorced from the if block).